Text/CustomTextClassifier/README.md

```diff
@@ -225,11 +227,142 @@ Finally, the indexer ties everything together. The indexer needs to be setup up
 }
 ```
 
+#### Query the index
+
+Now that we have a packed index, we can use the capabilities of the index to make useful queries. For example, using the sample data for scientific papers from the custom text classification documentation (found [here](https://github.com/Azure-Samples/cognitive-services-sample-data-files/tree/master/language-service/Custom%20text%20classification)), we can query the index to find all load agreements on a specific date. For example,
```

Comment:
"populated" rather than "packed"?

Reply:
I kept looking for "populated" for about 5 minutes. It only came to me after I pushed :).